doc/developer-guides/hld/hv-memmgt.rst

```diff
@@ -516,7 +516,7 @@ Host to Guest Mapping
 =====================
 
 ACRN hypervisor creates Service OS's host (HPA) to guest (GPA) mapping
-(EPT mapping) through the function ``prepare_vm0_memmap_and_e820()``
+(EPT mapping) through the function ``prepare_sos_vm_memmap_and_e820()``
```

Comment:
I can't find a function with this name, but there in one called ``prepare_sos_vm_memmap``

Reply:
OK, I will update.